app/controllers/service_uploads_controller.rb

```diff
@@ -10,7 +10,10 @@ def authorize_for_districtwide_access_admin
   end
 
   def create
-    service_upload = ServiceUpload.new(file_name: params['file_name'])
+    service_upload = ServiceUpload.new(
+      file_name: params['file_name'],
+      uploaded_by_educator_id: params['uploaded_by_educator_id']
```

Comment:
This is a security bug - it lets users tell us who they are in an untrusted way.  So if I'm logged in as myself, I can upload some services and tell the server "I'm Alex!" and it will say "okay, Alex uploaded this."  For tracking who is doing something, use `current_educator` in the controller instead of trusting the client.

Reply:
Ooooooh, good catch. Thanks, will fix.

Reply:
Darn untrustworthly client!!!